### PR TITLE
fix: update test-ng to version without bsh dependency that is vulnerable, org.apache-extras.beanshell:bsh is vulnerable to CVE-2016-2510, note, we do not use bsh in any runtime, it's test only

### PR DESCRIPTION
**Pull request by prb112**
fix: update test-ng to version without bsh dependency that is vulnerable, org.apache-extras.beanshell:bsh is vulnerable to CVE-2016-2510, note, we do not use bsh in any runtime, it's test only

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: conformance/fhir-ig-davinci-hrex/src/test/java/com/ibm/fhir/ig/davinci/hrex/test/ProfileTest.java [hunk 1]
```diff
@@ -53,4 +62,23 @@ public void testUSCoreValidation() throws Exception {
             throw e;
         }
     }
+
+    @Factory
```
Comment (lmsurpre) [inline conformance/fhir-ig-davinci-hrex/src/test/java/com/ibm/fhir/ig/davinci/hrex/test/ProfileTest.java:66]: how did it run before?

Comment (lmsurpre) [reply conformance/fhir-ig-davinci-hrex/src/test/java/com/ibm/fhir/ig/davinci/hrex/test/ProfileTest.java:66]: nm, now i see ProfileTestFactory.java (which is now removed). :-)

File: fhir-search/src/test/java/com/ibm/fhir/search/location/BoundingTest.java [hunk 2]
```diff
@@ -25,13 +25,13 @@ public void testBoundingBox() {
                 BoundingBox.builder().maxLatitude(10.0).minLatitude(-10.0).maxLongitude(20.0).minLongitude(-10.0)
                         .build();
         assertNotNull(bounding);
-        assertEquals(bounding.getMinLatitude(), -10.0);
-        assertEquals(bounding.getMinLongitude(), -10.0);
-        assertEquals(bounding.getMaxLatitude(), 10.0);
-        assertEquals(bounding.getMaxLongitude(), 20.0);
+        assertEquals(bounding.getMinLatitude(), Double.valueOf("-10.0"));
```
Comment (lmsurpre) [inline fhir-search/src/test/java/com/ibm/fhir/search/location/BoundingTest.java:28]: it doesn't autobox?

Comment (prb112) [reply fhir-search/src/test/java/com/ibm/fhir/search/location/BoundingTest.java:28]: It doesn't autobox anymore

File: conformance/fhir-ig-davinci-hrex/src/test/java/com/ibm/fhir/ig/davinci/hrex/test/ProfileTest.java (hunk 1 above)
Comment (prb112) [reply conformance/fhir-ig-davinci-hrex/src/test/java/com/ibm/fhir/ig/davinci/hrex/test/ProfileTest.java:66]: yeah - if it's outside the class it looses the scope

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**